### PR TITLE
docs(authn/x509): Detail clientAuth config for SSL.

### PR DESCRIPTION
**Pull request by stewchen**

### CODE REVIEW

File: setup/security/authentication/x509/index.md
Comment (dorbin) [inline setup/security/authentication/x509/index.md]: @stewchen , what does `WANT` add over not invoking this command at all?

File: setup/security/authentication/x509/index.md
Comment (dorbin) [inline setup/security/authentication/x509/index.md]: s/Setting/Set

File: setup/security/authentication/x509/index.md
Comment (dorbin) [inline setup/security/authentication/x509/index.md]: s/Configuring/Configure

File: setup/security/authentication/x509/index.md
Comment (dorbin) [inline setup/security/authentication/x509/index.md]: s/Enabling/Enable

**Review by dorbin — COMMENTED** [review]
LGTM, with those changes, and pending review by ttomsu

File: setup/security/authentication/x509/index.md
Comment (stewchen) [reply setup/security/authentication/x509/index.md]: `WANT` will allow the server to use a certificate if available, whereas when this is unset, the server will not support certificates at all.

**Comment by ttomsu** [comment]
LGTM

**Review by ttomsu — APPROVED** [review]

**Pull request merged**
